### ISSUE TITLE
I can't boot Steam OS when it was installed finished

### ISSUE THREAD

**Issue by qq1429358730** [holoiso-eol/holoiso#85]
hello! 
I entred "reboot" when steam os install finished, then it's won't start and I find my EFI floder was empty, I think the boot is not be installed.
thanks!

**Comment by qq1429358730** [comment]
My computer' CPU is AMD 5600X and GPU is AMD R9 fury

**Comment by theVakhovskeIsTaken** [comment]
Yeah, the EFI in current ISO is broken, i'll build fixed one later

**Comment by wynn1212** [comment]
For Temporarily fix. do this before executing `holoinstall` 
```
rm /usr/local/bin/holoinstall
wget https://raw.githubusercontent.com/theVakhovskeIsTaken/holoiso/3.0/airootfs/usr/local/bin/holoinstall -O /usr/local/bin/holoinstall 
chmod +x /usr/local/bin/holoinstall
```

**Comment by GuavaEater** [comment]
The temp fix above has not worked for me unfortunately, I can see that the disk does have a "Healthy (EFI System Partition)" of 224Mb but it doesn't appear on my Boot list. The commands themselves worked fine.

**Comment by wynn1212** [comment]
> The temp fix above has not worked for me unfortunately, I can see that the disk does have a "Healthy (EFI System Partition)" of 224Mb but it doesn't appear on my Boot list. The commands themselves worked fine.

Just wondering. Does mounting EFI partition shows any files inside that EFI files?

**Comment by GuavaEater** [comment]
> > The temp fix above has not worked for me unfortunately, I can see that the disk does have a "Healthy (EFI System Partition)" of 224Mb but it doesn't appear on my Boot list. The commands themselves worked fine.
> 
> Just wondering. Does mounting EFI partition shows any files inside that EFI files?

I'll see what I can do

**Comment by qq1429358730** [comment]
> > The temp fix above has not worked for me unfortunately, I can see that the disk does have a "Healthy (EFI System Partition)" of 224Mb but it doesn't appear on my Boot list. The commands themselves worked fine.
> 
> Just wondering. Does mounting EFI partition shows any files inside that EFI files?

Likewise, the fix has not work for me too, and EFI partition no ant files, and as shown in this figure, it looks like installation went wrong.
![wrong](https://user-images.githubusercontent.com/35194571/167623603-8f761d3a-3cd7-439f-b684-73ece103e36b.png)

**Comment by hopefullyusernamenk** [comment]
Yeah I have same problem. Do you think you will have installer fixed later today? Once o get home from school at about 4:30 i should be able to install it.

**Comment by Super-6-1** [comment]
The fix is here. 

https://github.com/theVakhovskeIsTaken/holoiso/issues/74#issuecomment-1121399627

as for the no such file or directory make sure your on a stable network connect. I had to redo the install a couple of times to get it to fully download everything.

**Comment by theVakhovskeIsTaken** [comment]
Fixed as of a8829508ccb02c8cf67978aa97806265167eee1f. Will publish ISO later

**Comment by GuavaEater** [comment]
> > > The temp fix above has not worked for me unfortunately, I can see that the disk does have a "Healthy (EFI System Partition)" of 224Mb but it doesn't appear on my Boot list. The commands themselves worked fine.
> > 
> > 
> > Just wondering. Does mounting EFI partition shows any files inside that EFI files?
> 
> I'll see what I can do

I think my issue was user error, there does not appear to be any EFI on the disk at all. I noticed after that I still had secure boot on, perhaps that prevented the changes from be written to the disk? I'll try again later this evening.
